modules/rosa-deleting-account-wide-iam-roles-and-policies.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-deleting-cluster.adoc
+// * rosa_getting_started_sts/rosa-sts-deleting-cluster.adoc
+
+ifeval::["{context}" == "rosa-sts-deleting-cluster"]
+:sts:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="rosa-deleting-account-wide-iam-roles-and-policies_{context}"]
+= Deleting the account-wide IAM roles and policies
+
+This section provides steps to delete the account-wide IAM roles and inline policies that you created for ROSA with STS deployments, along with the account-wide Operator policies. You can delete the account-wide AWS Identity and Access Management (IAM) roles and policies only after deleting all of the {product-title} (ROSA) with AWS Security Token Services (STS) clusters that depend on them.
+
+[IMPORTANT]
+====
+The account-wide IAM roles and policies might be used by other ROSA clusters in the same AWS account. You must only remove the roles if they are not required by other clusters.
+====
+
+.Prerequisites
+
+* You have installed a ROSA cluster.
+* You have installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your installation host.
+
+.Procedure
+
+. Delete the account-wide roles:
+.. List the account-wide roles in your AWS account by using the ROSA CLI (`rosa`):
++
+[source,terminal]
+----
+$ rosa list account-roles
+----
++
+.Example output
+[source,terminal]
+----
+I: Fetching account roles
+ROLE NAME                           ROLE TYPE      ROLE ARN                                                           OPENSHIFT VERSION
+ManagedOpenShift-ControlPlane-Role  Control plane  arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-ControlPlane-Role  4.10
+ManagedOpenShift-Installer-Role     Installer      arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Installer-Role     4.10
+ManagedOpenShift-Support-Role       Support        arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Support-Role       4.10
+ManagedOpenShift-Worker-Role        Worker         arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Worker-Role        4.10
+----
+.. Delete the account-wide roles:
++
+[source,terminal]
+----
+$ rosa delete account-roles --prefix <prefix> --mode auto <1>
+----
+<1> You must include the `--<prefix>` argument. Replace `<prefix>` with the prefix of the account-wide roles to delete. If you did not specify a custom prefix when you created the account-wide roles, specify the default prefix, `ManagedOpenShift`.
++
+[IMPORTANT]
+====
+The account-wide IAM roles might be used by other ROSA clusters in the same AWS account. You must only remove the roles if they are not required by other clusters.
+====
+
+. Delete the account-wide in-line and Operator policies:
```

Comment:
@arendej, @vkareh and @yuwang-RH hi! I have added this extra step and substeps to this procedure to include details on how to delete the account-wide in-line and Operator policies also. Please let me know if this looks good. Thanks!

Reply:
This looks fine, but doesn't the `rosa` CLI delete these already? cc @pvasant

Reply:
Thank you for confirming that, Priya. I will leave these steps in the PR.